src/ai/susi/server/api/cms/ListSkillService.java

```diff
@@ -200,16 +205,12 @@ public int compare(JSONObject a, JSONObject b) {
 
     }
 
-    ArrayList<String> listFilesForFolder(final File folder, ArrayList<String> fileList) {
-
+    private void listFilesForFolder(final File folder, ArrayList<String> fileList) {
```

Comment:
Since method is not invoked from any other package/class, therefore it should be a **private** method